.github/workflows/collection.yml

```
@@ -0,0 +1,46 @@
+name: Update collection
+on:
+  issue_comment:
+    types: [created]
+    branches:
+      - gh-pages
+jobs:
+  update_collection:
+    runs-on: ubuntu-latest
+    if: contains(github.event.comment.body, '#sign-off') && (github.event.comment.user.login == 'bamurtaugh' || contains(github.event.comment.user.login, 'devcontainers/maintainers'))
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '16'
```

```suggestion
          node-version: '20'
```

As 16 has reached EOL